docker/docker-compose-nightly.yaml

```diff
@@ -0,0 +1,38 @@
+# this file is not designed to be run directly
+# instead, use the docker-compose.<os>.<swift> files
+# eg docker-compose -f docker/docker-compose.yaml -f docker/docker-compose.1804.50.yaml run test
+
+services:
+
+  runtime-setup:
```

Comment:
I dont think we need to duplicate everything, this file can be reduced to one that just sets `FAST_TEST_RUN=false`. 

Then the docker compose command for the nightly job(s) can compose three files together, starting with the existing `docker-compose.yaml`, then this one, then the one that specifies the toolchain.

The last set env var will be used, so as long as `docker-compose-nightly.yaml` appears later in the docker compose command than `docker-compose.yaml` we should have `FAST_TEST_RUN` set to `false`.